opennms-alarms/daemon/src/main/java/org/opennms/netmgt/alarmd/AlarmPersisterImpl.java

```diff
@@ -211,7 +212,8 @@ private static void reduceEvent(OnmsEvent e, OnmsAlarm alarm, Event event) {
                         alarm.setSeverity(OnmsSeverity.valueOf(e.getSeverityLabel()));
                     } else if (fieldName.toLowerCase().contains("descr")) {
                         alarm.setDescription(e.getEventDescr());
-                        alarm.setSeverity(OnmsSeverity.valueOf(e.getSeverityLabel()));
```

Comment:
Was this removal intentional?

Reply:
I agree with @pbrane that this one is likely a copypasta bug.